physical_property/permeability/permeability_challenge_instructions.md

```diff
@@ -53,6 +53,19 @@ The concentration values from the Donor and Acceptor compartment are used in the
 
 ![permeability equations](../images/permeability_equations.jpeg)
 
+
+These experiments used the “Corning GentestTM pre-coated PAMPA plate system,” with a “lipid-oil-lipid” trilayer in the artificial membrane. The artificial membrane was composed of 40 µg of DOPC on both sides of the plate with a 1 µL deposit of hexadecane in the center, as indicated [here](experimental_resources/Application-Note-475.pdf).
+
+One problem is that we’re not sure how close this artificial bilayer is to a pure lipid bilayer. Some groups chose to use an unstressed DOPC bilayer for calculations.  However, there may be regions of non-lamellar lipids (as shown by Assmus et al. [3](experimental_resources/molpharmaceut.2017.pdf) or the surface area of the DOPC could be substantially changed by the formulation.
```

Comment:
Maybe add at end of paragraph "This is an active topic of research and one where further work is needed."

Reply:
Added.